tamr_unify_client/models/project/estimated_pair_counts.py

```diff
@@ -20,15 +20,15 @@ def is_up_to_date(self) -> bool:
     def total_estimate(self) -> dict:
         """The total number of estimated candidate pairs and generated pairs for the model across all clauses.
 
-        :type: dict
+        :type: dict[str, int]
         """
         return self._data.get("totalEstimate")
 
     @property
     def clause_estimates(self) -> dict:
         """The clause name, candidate pair count, and generated pair count for each clause in the model.
 
-        :type: dict
+        :type: dict[dict[str, int]]
```

Comment:
Same comment about adding a description for what the `dict` entries conceptually represent. Also, `dict[dict[str, int]]` doesn't quite make sense since the outer `dict` should be described by 2 types (corresponding to the key/value types) where as right now it just has 1 inner type of `dict[str, int]`